### PR TITLE
add python2 and python3 support

### LINKED ISSUE #10: python 3.6 default image / buildspec.yml

### LINKED ISSUE #11: Fix python default buildspec
pip install command needs to set the installation destination (e.g. `-t /tmp/vendored`) in a non-default location so that the files can be zipped into build artifacts.

### PR DESCRIPTION
**Pull request by vincentsarago**
This PR add support for python2.7 and python3.6 env

closes https://github.com/rclark/bundle-shepherd/issues/10
closes https://github.com/rclark/bundle-shepherd/issues/11

cc @rclark

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -130,10 +130,11 @@ Both fields are optional, and if the default values are fine, you need not inclu
...
 ### image
 
-There are two default images provided by bundle-shepherd:
+There are three default images provided by bundle-shepherd:
 
 - `nodejs6.x` (default)
 - `python2.7`
+- `python3.6`
 
 If you select either of these, bundle-shepherd will build your `.zip` file using its default images and build instructions for that runtime. You may also specify the URI of any other Docker image. If you choose to do so, you will have to also provide your own `buildspec.yml`.
```
Comment (rclark) [inline readme.md]: This should change to `If you select any of these, bundle-shepherd...`

File: buildspecs/python2.7.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
-version: 0.1
+version: 0.2
 phases:
   pre_build:
     commands:
-      - pip install -r ./requirements.txt
+      - pip install -r ./requirements.txt -t /tmp/vendored
       - rm -rf .git/
+      - cp -r . /tmp/vendored
```
Comment (rclark) [inline buildspecs/python2.7.yml]: What is the benefit of copying everything to this other directory? Would an alternative be to set the `-t` flag on `pip install` to the `${pwd}`?

**Review by rclark — COMMENTED** [review]
Thank you!!

File: buildspecs/python2.7.yml (hunk 2 above)
Comment (vincentsarago) [reply buildspecs/python2.7.yml]: oh this is true I didn't realized it 👍

**Review by rclark — APPROVED** [review]
:+1: thanks @vincentsarago

**Pull request merged**